packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -267,9 +275,12 @@ Future<Map<String, dynamic>> collect(Uri serviceUri, Set<String>? libraryNames,
   @visibleForTesting bool forceSequential = false,
   @visibleForTesting FlutterVmService? serviceOverride,
   bool branchCoverage = false,
+  required Map<String, Set<int>> coverableLineCache,
```

Comment:
@liamappelbe marking this required is a breaking change, and will be difficult to get into Google. Can we mark it as optional for the short term?